src/porepy/grids/boundary_grid.py

```diff
@@ -11,7 +14,7 @@
 import porepy as pp
 
 
-class BoundaryGrid(pp.Grid):
+class BoundaryGrid:
```

Comment:
I agree this is a fair simplification for now. Then let's revisit once we start using this grids in earnest in the models.